documentation/operators/src/nodes/bonding.md

```diff
@@ -0,0 +1,68 @@
+# Bonding Nym Node
+
+```admonish caution
+If you unbond your Nym Node that means you are leaving the mixnet and you will lose all your delegations (permanently). You can join again with the same identity key, however, you will start with **no delegations**.
+```
+
+Nym Mixnet operators are rewarded for their work every epoch (60 minutes). To prevent centralisation, [Nym API](nym-api.md) is ran by distributed validators on Nyx blockchain.
```

Comment:
Are we actually spreading out which api instances we're hitting on client startup yet?

Reply:
I think those can be seen in the swagger and in harbour master, both mentioned on top of nym-node page. Should it be metioned here as well?